example-app/src/main/cpp/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ project("myapplication")
 # or SHARED, and provides the relative paths to its source code.
 # You can define multiple libraries, and CMake builds them for you.
 # Gradle automatically packages shared libraries with your APK.
-set(CMAKE_CXX_FLAGS -fno-omit-frame-pointer)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-omit-frame-pointer -std=c++11")
```

Comment:
We should use [`target_compile_options`](https://cmake.org/cmake/help/latest/command/target_compile_options.html) instead. Assuming that we target CMake versions that support it, but I think we do.